Cocktails9/Cocktails9/View/BottomRow/BottomRowView.swift

```diff
@@ -37,15 +37,24 @@ struct BottomRowView: View {
         NavigationStack {
             ZStack(alignment: .bottom){
                 TabView(selection: $selectedTab) {
-                    HomeView()
-                        .tag(0)
-                    
-                    FavoritesView()
-                        .tag(1)
-                    
-                    ProfileView()
-                        .tag(3)
+                    Group {
+                        //HomeView()
+                        DrinksGrid()
+                            .background(Color.green.opacity(0.2))
+                            .tag(0)
+                        
+                        FavoritesView()
+                            .background(Color.green.opacity(0.2))
+                            .tag(1)
+                        
+                        ProfileView()
+                            .tag(2)
+                    }
+//                    .toolbarBackground(.clear, for: .tabBar)
+                        .toolbarBackground(.hidden, for: .tabBar)
+//                        .toolbarColorScheme(.dark, for: .tabBar)
```

Comment:
remove commented code
```suggestion
                        .toolbarBackground(.hidden, for: .tabBar)
```